frontend/README.md

```diff
@@ -18,4 +18,5 @@ If you require any assistance or support, feel free to let us know. Join our Sla
 Ohha! The participation list is totally empty. Hurry up if your want to have your name
 here before anyone else.
 
-- Example project by @GITHUB_USER_HANDLE. Link.
+- https://github.com/hobaDevHome/moviesApp
```

Comment:
We can add the project name, your github handle, links to github & the live app. 

```suggestion
- Movies app by @hobaDevHome. [Github Link](https://github.com/hobaDevHome/moviesApp). [Live Link](https://movisearchapp.netlify.app/). 
```